examples/TestClient.java

```diff
@@ -0,0 +1,526 @@
+			System.out.println(x.getMessage());
+		}
+		// Quote(POST)
+		System.out.println("Service::Quote(POST)");
+		try	{
+			List<TranslationJob> qt_jobs = new ArrayList<TranslationJob>();
+			qt_jobs.add(new TranslationJob("Test job for quoting.(1) (" + this.sessionHash + ")", "Test.",
+					"en", "ja", Tier.STANDARD));
+			qt_jobs.add(new TranslationJob("Test job for quoting.(2) (" + this.sessionHash + ")", "Test.",
+					"en", "ja", Tier.PRO));
+			response = this.gc.determineTranslationCostText(qt_jobs);
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		System.out.println("[testService] end@" + this.sessionHash);
+	}
+	
+	/**
+	 * Test jobs functionalities.
+	 */
+	public void testJobs(Boolean verbose) {
+		verbose = verbose == null ? true : verbose;
+		System.out.println("[testJobs] start@" + this.sessionHash);
+		JSONObject response = null;
+		List<TranslationJob> pll;
+		// Jobs(POST) single
+		try	{
+			System.out.println("Jobs::(POST) single");
+			pll = Arrays.asList(
+					new TranslationJob("[Single] en2ja::" + this.sessionHash, "Test.",
+							"en", "ja", Tier.STANDARD)
+					);
+			response = this.gc.postTranslationJobs(pll, false);
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		// Jobs(POST) multiple
+		try	{
+			System.out.println("Jobs::(POST) multiple");
+			pll = Arrays.asList(
+					new TranslationJob("[Multiple] en2ja(1)::" + this.sessionHash, "Test. (1)",
+							"en", "ja", Tier.STANDARD),
+					new TranslationJob("[Multiple] en2ja(2)::" + this.sessionHash, "Test. (2)",
+							"en", "ja", Tier.STANDARD)
+					);
+			response = this.gc.postTranslationJobs(pll, true);
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		// Jobs(GET)
+		try	{
+			System.out.println("Jobs::(GET)");
+			response = this.gc.getTranslationJobs();
+			System.out.print("[OK]");
+			System.out.println(verbose ? response.toString() : "");
+		} catch (GengoException x) {
+			System.out.print("[NG]");
+			System.out.println(x.getMessage());
+		}
+		// Jobs(GET) by id
+		Integer[] jobs_get = null;
+		try	{
+			jobs_get = new Integer[(int) Math.floor(response.getJSONArray("response").length() / 2)];
+			for (int i = 0; i < (int) Math.floor(response.getJSONArray("response").length() / 2); i++) {
+				jobs_get[i] = response.getJSONArray("response").getJSONObject(i).getInt("job_id");
+			}
+		} catch (JSONException x) {
+				System.out.print("[NG]");
+				System.out.println(x.getMessage());
+		}
+		try	{
+			System.out.println("Jobs::(GET) by id");
+			response = this.gc.getTranslationJobs(Arrays.asList(jobs_get));
```

Comment:
I don't understand what you want to tell. You mean `Arrays.asList()` should take `null` as argument? Or the `jobs_get` argument will must be `null` all the time because the `try` block which is Line#186 - #189 is not working properly?